pkg/validation/exception/validate.go

```diff
@@ -5,10 +5,22 @@ import (
 
 	"github.com/go-logr/logr"
 	kyvernov2alpha1 "github.com/kyverno/kyverno/api/kyverno/v2alpha1"
+	"github.com/kyverno/kyverno/pkg/webhooks"
+)
+
+const (
+	namespacesDontMatch = "PolicyException resource namespace must match the defined namespace."
+	disabledPolex       = "PolicyException resources would not be processed until it is enabled."
 )
 
 // Validate checks policy exception is valid
-func Validate(ctx context.Context, logger logr.Logger, polex *kyvernov2alpha1.PolicyException) error {
+func Validate(ctx context.Context, logger logr.Logger, polex *kyvernov2alpha1.PolicyException, po webhooks.ExceptionOptions) (error, []string) {
```

Comment:
@eddycharly Validate would need to import /webhooks/exception if `ExceptionOptions{}` is moved near `handlers{}` (which is in /webhooks/exception )

Reply:
Ok, maybe we need to move it somewhere else then.